Core/include/Acts/Vertexing/IterativeVertexFinder.hpp

```diff
@@ -105,6 +105,7 @@ class IterativeVertexFinder {
     bool doMaxTracksCut = false;
     int maxTracks = 5000;
     double cutOffTrackWeight = 0.01;
+    double cutOffTrackWeightReassign = 1;
```

Comment:
I would suggest to add some documentation, especially with this double weight cut off, where the second is used to decide if the track will be reassigned.

Reply:
```suggestion
    /// If `reassignTracksAfterFirstFit` is set this threshold will be used to decide if a track should be checked for reassignment to other vertices
    double cutOffTrackWeightReassign = 1;
```